raiden/udp_message_handler.py

```diff
@@ -89,7 +89,7 @@ def handle_message_refundtransfer(raiden: 'RaidenService', message: RefundTransf
         )
     else:
         state_change = ReceiveTransferRefund(
-            routes,
+            message.sender,
```

Comment:
Aren't the routes provided to the state machine via the `ActionInitXXX` type of state changes?

Reply:
every state change that requires a new mediated message must provide the routes. it's done like this because we need to account for the path finding service communication.